builtin/providers/aws/resource_aws_instance.go

```diff
@@ -129,9 +129,11 @@ func resourceAwsInstance() *schema.Resource {
 			},
 
 			"network_interface_id": &schema.Schema{
-				Type:     schema.TypeString,
-				Computed: true,
-				Optional: true,
+
+				Type:          schema.TypeString,
```

Comment:
Why not the list? Doesn't look like a whole lot more code. Another question is if the tf config changes this value, should the instance be rebuilt?  That is, should `ForceNew: true`